metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/histogram_data_point.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Aggregation
+        HistogramDataPoint = Struct.new(:attributes,            # optional Hash{String => String, Numeric, Boolean, Array<String, Numeric, Boolean>}
+                                        :start_time_unix_nano,  # optional Integer nanoseconds since Epoch
```

Comment:
Is `start_time_unix_nano` really optional? How would we end up with it missing?

Reply:
It is really option in the OTLP protos

```
  // StartTimeUnixNano is optional but strongly encouraged, see the
  // the detailed comments above Metric.
  //
  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January
  // 1970.
  fixed64 start_time_unix_nano = 2;
```
https://github.com/open-telemetry/opentelemetry-proto/blob/v0.19.0/opentelemetry/proto/metrics/v1/metrics.proto#L391-L396

```
// # TimeUnixNano
//
// This field is required, having consistent interpretation across
// DataPoint types.  TimeUnixNano is the moment corresponding to when
// the data point's aggregate value was captured.
//
// Data points with the 0 value for TimeUnixNano SHOULD be rejected
// by consumers.
//
// # StartTimeUnixNano
//
// StartTimeUnixNano in general allows detecting when a sequence of
// observations is unbroken.  This field indicates to consumers the
// start time for points with cumulative and delta
// AggregationTemporality, and it should be included whenever possible
// to support correct rate calculation.  Although it may be omitted
// when the start time is truly unknown, setting StartTimeUnixNano is
// strongly encouraged.
```

https://github.com/open-telemetry/opentelemetry-proto/blob/v0.19.0/opentelemetry/proto/metrics/v1/metrics.proto#L144-L161

Reply:
```suggestion
                                        :start_time_unix_nano,  # Integer nanoseconds since Epoch
```